neo/Prompt/Commands/SC.py

```diff
@@ -299,8 +318,8 @@ def command_desc(self):
                            f"the return type of the smart contract output\n\n"
                            f"{' ':>17} For more information about parameter types see\n"
                            f"{' ':>17} https://neo-python.readthedocs.io/en/latest/data-types.html#contractparametertypes\n", optional=True)
-
-        params = [p1, p2, p3, p4, p5, p6]
+        p7 = ParameterDesc('--fee', 'fee to give your transaction priority (> 0.001) e.g. --fee=0.01', optional=True)

```

Comment:
```suggestion
        p7 = ParameterDesc('--fee', 'Attach GAS amount to give your transaction priority (> 0.001) e.g. --fee=0.01', optional=True)
```
just saying "fee" doesn't tell the user where the fee is coming from. They have to derive it's GAS from the fact that NEO is indivisible. This suggestion clarifies it.

Reply:
I copied this verbage from the other locations we use a fee such as https://github.com/CityOfZion/neo-python/blob/e561e5b57d4093a9ee0c87a8312d4a4d16f6fd5e/neo/Prompt/Commands/Send.py#L44
Should I change it everywhere?

Reply:
Yes please :)